libs/core/src/EventReader.cpp

```diff
@@ -21,14 +21,30 @@ EventReader::EventReader() {
   string inputFilePath;
   config.GetValue("inputFilePath", inputFilePath);
 
+  string custom_redirector = "";
+  try {
+    config.GetValue("redirector", custom_redirector);
+  } catch (const Exception &e) {
+    warn() << "Couldn't read redirector from config file" << endl;
```

Comment:
This is a general comment: I feel like there are too many warnings now, so maybe for things that are perfectly fine (like not specifying a custom redirector) we should just print `info`, saying "No custom redirector found in the config file - will you default", or something like this?